src/main/java/com/example/demo/matcher/MatcherController.java

```diff
@@ -27,57 +26,55 @@ public class MatcherController {
     @Autowired
     private final TradeService tradeService;
 
-    @GetMapping(value = "/orderbook/buy")
+    @GetMapping(value = "/public/orderbook/buy")
     public ResponseEntity<List<OrderbookItem>> orderbook_buy() {
         return ResponseEntity.ok(orderService.getOrderbook(OrderAction.BUY));
     }
 
-    @GetMapping(value = "/orderbook/buy/{accountId}")
-    public ResponseEntity<List<OrderbookItem>> orderbook_buy(@PathVariable String accountId) {
+    @GetMapping(value = "/private/orderbook/buy/{username}")
+    public ResponseEntity<List<OrderbookItem>> orderbook_buy(@PathVariable String username) {
         try {
-            UUID accountUuid = UUID.fromString(accountId);
-            return ResponseEntity.ok(orderService.getOrderbook(OrderAction.BUY, UUID.fromString(accountId)));
+            return ResponseEntity.ok(orderService.getOrderbook(OrderAction.BUY, username));
         }
         catch(Exception e) {
             return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
         }
     }
 
-    @GetMapping(value = "/orderbook/sell")
+    @GetMapping(value = "/public/orderbook/sell")
```

Comment:
So this one won't be authenticated? Is that why you're specifying it's public?

Reply:
Yup
That way it's clear to the person programming the client that they don't need a token.
But also it makes it easy for me to configure which endpoints are authenticated and which aren't!